fili-core/src/main/java/com/yahoo/bard/webservice/logging/RequestLog.java

```diff
@@ -308,15 +277,15 @@ public static void stopTiming(String timePhaseName) {
             LOG.warn("Tried to stop non-existent phase: {}", timePhaseName);
             return;
         }
-        timePhase.stop();
+        timePhase.close();
```

Comment:
`close` has to do the registering anyway, because that's the method invoked when the `try-with-resources` block ends. So even if we didn't change how `close` works, and had `close` delegate to `stop`, we still wouldn't need to change this invocation.